src/js/components/form-elements/TableBodyVirtualized.jsx

```diff
@@ -38,7 +38,14 @@ class TableBodyVirtualized extends Component {
       }
     } else {
       _.forEach(fields.value, (field) => {
+        const dynamicAttr = getDynamicRowAttr ?
+          getDynamicRowAttr({ ...properties, rowValues: field }) : {};
         const subfields = field[subfieldKey];
+
+        if (dynamicAttr.hideRow) {
+          return;
```

Comment:
I think you should return 0 here

Reply:
@pmuchowski this is lodash's forEach version of `continue`